File: tingeltangel-cli/src/main/java/tingeltangel/TingeltangelCli.java
package tingeltangel;

import tingeltangel.cli_ng.CLI;

public class TingeltangelCli {
    public static void main(String[] args) {
        Tingeltangel.initialize();
        CLI.init();
        if (args.length > 0) {
            CLI.run(args[0]);
        } else {
            CLI.run("");
        }
    }
}

File: tingeltangel-cli/src/main/java/tingeltangel/TingeltangelCli.kt
package tingeltangel

import tingeltangel.cli_ng.CLI

object TingeltangelCli {
    @JvmStatic
    fun main(args: Array<String>) {
        Tingeltangel.initialize()
        CLI.init()
        CLI.run(
            if (args.isNotEmpty()) {
                args[0]
            } else ""
        )
    }
}
